python_modules/dagster/dagster/_core/storage/upath_io_manager.py

```diff
@@ -52,26 +51,53 @@ def get_metadata(
         """Child classes should override this method to add custom metadata to the outputs."""
         return {}
 
+    # Read/write operations on paths can generally be handled by methods on the
+    # UPath class, but when the backend requires credentials, this isn't
+    # always possible. Override these path_* methods to provide custom
+    # implementations for targeting backends that require authentication.
+
+    def path_rm(self, path: UPath) -> None:
+        """Remove the file or object at the provided path."""
+        path.unlink()
+
+    def path_exists(self, path: UPath) -> bool:
+        """Check if a file or object exists at the provided pat.."""
+        return path.exists()
+
+    def path_mkdir(self, path: UPath):
```

Comment:
Nitpick: `-> None`

Reply:
Also, why not just `mkdir`?

Reply:
in the spirit of your above comment on abbrevations, how about `make_directory`

Reply:
Sounds good